codemods/remove-context-provider/.codemodrc.json

```diff
@@ -0,0 +1,13 @@
+{
+	"version": "1.0.0",
+	"name": "react/19/remove-context-provider",
+	"private": false,
+	"engine": "jscodeshift",
+	"meta": {
+		"tags": ["react"],
+		"useCaseCategory": "migration"
+	},
+	"applicability": {
+		"from": [["react", "<=", "^18.0.0"]]
```

Comment:
it's set to less than latest minor version of v18. ^18.0.0 is a semver for that. in this example, it's saying from react v0 to react v18 (latest minor version of 18th major). of course, there is no guarantee that it works for react < v16, but i decided not to add another border, as it might work for different older versions, depending on how they are being used.